portfolio/src/main/webapp/index.html

```diff
@@ -81,10 +80,27 @@ <h2>Hobbies</h2>
       <p>Click here to get random facts about me!:</p>
       <button onclick="addRandomGreeting()">Click me!</button>
       <div id="greeting-container"></div>
+    </div>
+
+    </br>
```

Comment:
There is no such tag `</br>` - either use `<br />` or `<br>` (or just delete it if you don't really need it). Same comment throughout this file.

Reply:
Got it, I'll replace them to <br />